general/server/media/comics.md

```diff
@@ -0,0 +1,45 @@
+---
+uid: server-media-comics
+title: Comics
+---
+
+# Comics / Mangas
+
+Comics and mangas, from now on referred to as comics, should usually be in the library root directory or in a subfolder for the individual comics. The subfolders allow for organization of metadata and images.
+
+For best results, it is recommended to name the files similar to the following schema where the volume number and issue number are used **interchangeable**.
+
+```txt
+Series_Name #IssueNumber (of Count) (PubYear).ext
```

Comment:
Would this look something like the following?

```txt
Comics
├── Plastic Man #002 (of 1000) (1944).cbz
```

Reply:
For example. Another example would be `Attack on Titan #003 (2012).cbz` where I left out the `(of Count)` part as the series was still on-going when I named them. It's basically one of the name schemes from comictagger as I don't think it's necessary to re-invent the wheel here. ;)